android/app/src/main/java/org/fossasia/openevent/dbutils/RealmDatabaseMigration.java

```diff
@@ -2,10 +2,11 @@
 
 import io.realm.DynamicRealm;
 import io.realm.RealmMigration;
+import io.realm.RealmSchema;
 
 public class RealmDatabaseMigration implements RealmMigration {
 
-    public static final long DB_VERSION = 0;
+    public static long DB_VERSION = 0;
```

Comment:
outside the if statement? 
but if the value changes why do we create it as final?
sorry not getting your point.